.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,42 +1,19 @@
-# All SDK Contribution checklist:
+# Contributing to the Azure SDK
 
-This checklist is used to make sure that common guidelines for a pull request are followed.
-- [ ] **Please open PR in `Draft` mode if it is:**
-	- Work in progress or not intended to be merged.
-	- Encountering multiple pipeline failures and working on fixes.
-- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
-- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
-- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**
+Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.
 
-### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
-- [ ] Title of the pull request is clear and informative.
-- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
+## Pull Request etiquette and best practices
 
-### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#testing-guidelines)
-- [ ] Pull request includes test coverage for the included changes.
+### Reviewers
```

Comment:
This calls out all the things one shouldn't do... maybe we should also add things around approvals -- when should one approve? Something along the line of requesting approval from the area owner?

Reply:
>  Something along the line of requesting approval from the area owner?

Are there scenarios where this wouldn't be automatically done via codeowners?

Reply:
I don't think there is a GitHub way to block on area owners. I know that Fabric Bot has an awesome PR flow where you can request (and the bot will block) PRs from certain people but we're not using that yet.